src/libs/common.go

```diff
@@ -163,13 +166,42 @@ func (i *cfgArray) Set(str string) error {
 func CheckCommandLineConfig() {
 	var cmdlineCfg cfgArray
 
+	pprofflag := flag.Bool("pprof", false, "enable pprof")
 	version1 := flag.Bool("v", false, "print version and exit")
 	version2 := flag.Bool("version", false, "print version and exit")
 	flag.Var(&cmdlineCfg, "cfg", "Configuration key=val")
 
 	configFilePath := flag.String("config-path", "conf/", "conf/")
 	flag.Parse()
 
+	// Reset default routes (removing access to profiling)
+	http.DefaultServeMux = http.NewServeMux()
+
+	// Recreating routes to profiling manually
+	pprofServeMux := http.NewServeMux()
```

Comment:
Can we add all the pprof http handling as part of pprofflag ?

Reply:
Is it better now?